src/load_balancer.py

```diff
@@ -0,0 +1,80 @@
+from abc import abstractmethod
+from typing import Protocol
+
+import aioredis
+
+
+class LockService(Protocol):
+    """Load balancer interface based on Redis."""
+
+    @abstractmethod
+    async def acquire_lock(self, chat_id: int) -> None:
+        """Set the lock on operations with a specific chat.
+        If the key 'chat_id' exists in the DB, the lock is acquired on the chat.
```

Comment:
What is a DB in the context of an interface? Speak in terms of the behavior, not particular implementation.